src/Catel.Analyzers.Tests/CTL0003/CTL0003DiagnosticFacts.cs

```diff
@@ -231,6 +231,43 @@ private bool FilterCondition(object obj)
 
                 Solution.Verify<MethodsAnalyzer>(analyzer => RoslynAssert.NoAnalyzerDiagnostics(analyzer, Descriptors.CTL0003_FixOnPropertyChangedMethodToMatchSomeProperty, before));
             }
+
+            [TestCase]
+            public void Valid_Code_Event_Handler()
+            {
+                var before = @"
+namespace TestApp1
+{
+    using Catel.MVVM;
+    using System.ComponentModel.DataAnnotations;
+    using System.Windows.Data;
+
+    public class DummyTestViewModel : ViewModelBase
+    {
+        public DummyTestViewModel(IDummyConditions dummyConditions)
+        {
+            dummyConditions.DummyConditionChanges += OnDummyConditionChanged;
```

Comment:
maybe the event itself should also be DummyConditionChanged, but I think it's irrelevant here